Tests/AsyncAlgorithmsTests/TestChannel.swift

```diff
@@ -10,268 +10,167 @@
 //===----------------------------------------------------------------------===//
 
 import XCTest
-import AsyncAlgorithms
+@testable import AsyncAlgorithms
```

Comment:
Do we still need the testable import? I think we are only using public API by now

Reply:
nope. good catch. this is fixed.